### PR TITLE
Fetch tokens much faster (and other fixes)

### PR DESCRIPTION
**Pull request by nop33**
Moreover, it refers to issues #171 , #168, #154

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "@alephium/sdk": "0.7.2",
-    "@alephium/web3": "^0.13.0",
+    "@alephium/sdk": "0.7.3",
```
Comment (mvaivre) [inline package.json:19]: We can use 0.7.5 now.

Comment (nop33) [reply package.json]: Updated!

**Comment by nop33** [comment]
Rebased to fix conflicts.

**Review by mvaivre — APPROVED** [review]
LGTM!

**Pull request merged**